### PR TITLE
Assign and Remove Collaborators from the todo

### PR DESCRIPTION
**Pull request by vrrayz**
This endpoint enables the user to add or remove collaborators from a todo

### CODE REVIEW

File: app/Http/Controllers/AssignUserController.php [hunk 1]
```diff
@@ -47,4 +61,41 @@ public function assign(Request $request, $todoId)
 
         // return response()->json($response);
     }
+    public function remove(Request $request, $todoId)
```
Comment (kayleb01) [inline app/Http/Controllers/AssignUserController.php:64]: @vrrayz Its like you are not aware  of  the current structure of the Todo, Please contact @Ericogia  to give you the updated structure, we only have a collaborators array in which we add users

**Review by kayleb01 — CHANGES_REQUESTED** [review]
I just checked @chixmxkx  already worked and submitted this and her sumission has been approved. Sorry man

**Comment by kayleb01** [comment]
I just checked @chixmxkx  already worked and submitted this and her sumission has been approved. Sorry man

File: app/Http/Controllers/AssignUserController.php (hunk 1 above)
Comment (vrrayz) [reply app/Http/Controllers/AssignUserController.php:64]: > I just checked @chixmxkx already worked and submitted this and her sumission has been approved. Sorry man

She worked on removing users from a Todo. I added the option of removing teams from the Todo. Check it well

**Comment by vrrayz** [comment]
@kayleb01 please check my comment

**Pull request closed without merging**